packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -190,11 +196,11 @@ class Metrics {
       origin: {
         buildType: 'prod',
         name: 'endpoint',
-        networkType: 'unknown',
+        networkType: 'unknown', // TODO: check if we can get mediaType
         userAgent: this.getUserAgentInfo(),
         clientInfo: {
-          clientType: options.clientType,
-          os: 'linux', // TODO: BUG AND DOESNT ACCEPT OSNAME ON CA -> bowser.osname || 'null'
+          clientType: options.clientType, // TODO: Only clientType: 'TEAMS_CLIENT' is whitelisted
+          os: 'linux' || this.osName(), // TODO: For some reason Mqa does not accept mac
```

Comment:
What is the purpose of the OR statement here? If we can't send `mac` why even have the `this.osName()`? Also it is out of order and `this.osName()` will never be called.

Reply:
just added it for future data but ya can remove it